Doc/pages/H_conv.rst

```diff
@@ -166,3 +166,107 @@ long trajectories. The argon trajectory used contain a total of 2048
 atoms which appears to be sufficient to obtain enough statistics for
 good static structure factor results, so that only a short MD
 simulation would be required.
+
+
+Simulation Time Step
+~~~~~~~~~~~~~~~~~~~~
+
+Dynamic Incoherent Structure Factor
+-----------------------------------
+The DISF (also the DCSF) calculation probes the dynamics of the MD trajectory at
+different time scales. For example, smaller values of :math:`q`
```

Comment:
If you don't want to keep writing 
```
:math:`q`
```

Do you know about ReST [replace](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#substitutions)? 

```
.. |q| replace:: :math:`q`
```
Means you could write this as:
```suggestion
different time scales. For example, smaller values of |q|
```

Probably not necessary now this is written, but good to know.

Reply:
Let's leave it for now. I might use it next time I update the docs.